include/SABER/SaberCondAllocator.h

```diff
@@ -144,10 +144,54 @@ class SaberCondAllocator
         return negConds.test(id);
     }
 
-    /// Get dominators
-    inline DominatorTree* getDT(const Function* fun)
+    inline const bool pdtDominates(const BasicBlock* bbKey, const BasicBlock* bbValue)
     {
-        return cfInfoBuilder.getDT(fun);
+        return dominates(bbKey,bbValue,SVFFunction::DtK::PostDominatorTree);
+    }
+
+    inline const bool dtDominates(const BasicBlock* bbKey, const BasicBlock* bbValue)
+    {
+        return dominates(bbKey,bbValue,SVFFunction::DtK::DominatorTree);;
+    }
+
+    inline const bool dominates(const BasicBlock* bbKey, const BasicBlock* bbValue, const SVFFunction::DtK dtType)
```

Comment:
Put the function body into two different dominates, i.e., pdtDominates and dtDominates. Do not mix them